requirements/dev.txt

```diff
@@ -5,7 +5,6 @@ pytest-cov~=2.10.1
 pytest-xdist~=2.5
 pytest-env~=0.6.2
 pytest-rerunfailures~=9.1.1
-pylint~=2.15.0
 pyyaml~=5.4
 ruff==0.0.237  # loose the requirement once it is more stable
```

Comment:
Remove the pin? So always latest in CI.

Reply:
why not

Reply:
it doesn't hurt

Reply:
well it shows its annoyance as 238 introduced more rules.